WordPress/WordPressShare/PostStatusPickerViewController.swift

```diff
@@ -0,0 +1,98 @@
+import Foundation
+import UIKit
+import WordPressShared
+
+class PostStatusPickerViewController : UITableViewController
+{
+    // MARK: - View Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+        setupTableView()
+        setupNoResultsView()
+        loadStatuses()
+    }
+    
+    
+    // MARK: - UITableView Methods
+    override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return statuses.count
+    }
+    
+    override func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
+        return rowHeight
+    }
+    
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        var cell = tableView.dequeueReusableCellWithIdentifier(reuseIdentifier)
```

Comment:
We really can't (or shouldn't) because `WPTableViewCellSubtitle` lives within `WPImmuTableCells`, and there are few things tied up to ImmuTable. Which is not really in use into the extension, and would be neat to keep the footprint as minimum as possible.

Maybe we could/should detach WPTableViewCellSubtitle from ImmuTable instead (?)